packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -689,6 +698,40 @@ class CameraController extends ValueNotifier<CameraValue> {
     }
   }
 
+  /// Set the video stabilization mode for the selected camera.
+  ///
+  /// On Android (when using camera_android_camerax) and on iOS
+  /// the supplied [mode] value should be a mode in the list returned
+  /// by [getVideoStabilizationSupportedModes].
+  ///
+  /// Throws a [CameraException] when a not supported video stabilization
+  /// mode is supplied.
+  Future<void> setVideoStabilizationMode(VideoStabilizationMode mode) async {
+    _throwIfNotInitialized('setVideoStabilizationMode');
+    try {
+      await CameraPlatform.instance.setVideoStabilizationMode(_cameraId, mode);
+      value = value.copyWith(videoStabilizationMode: mode);
+    } on PlatformException catch (e) {
+      throw CameraException(e.code, e.message);
+    }
+  }
+
+  /// Gets a list of video stabilization modes that are supported for the selected camera.
+  ///
+  /// Will return the list of supported video stabilization modes
+  /// on Android (when using camera_android_camerax package) and
+  /// on iOS. Will return an empty list on all other platforms.
```

Comment:
> Will return an empty list on all other platforms.

This is no longer true. It throws an `UnimplementedError` on other platforms.

Reply:
Just fixed it. Thanks!